src/combat.h

```diff
@@ -41,7 +41,7 @@ class TargetCallback final : public CallBack
 
 struct CombatParams
 {
-	std::forward_list<std::unique_ptr<const Condition>> conditionList = {};
+	std::forward_list<std::shared_ptr<const Condition>> conditions = {};
```

Comment:
Let's change this one to a vector (preferred) or a deque, forward list is wasteful

Reply:
Right, we also have places like `Player::getMuteConditions()` and `Player {storedConditionList}`. Should we change those as well?